rover_auxiliary/src/Antenna/antenna_node.cpp

```diff
@@ -0,0 +1,134 @@
+#include "antenna_node.hpp"
+#include <rover_lib2/helpers/constants.hpp>
+#include <fstream>
+#include <sstream>
+
+namespace
+{
+    rover_msgs::msg::AntennaStatus toRosMsg(const sAntennaMsg& msg_)
+    {
+        rover_msgs::msg::AntennaStatus rosMsg;
+        rosMsg.connected = msg_.connected;
+        rosMsg.rssi = msg_.rssi;
+        rosMsg.rxrate = msg_.rxRate;
+        rosMsg.txrate = msg_.txRate;
+        return rosMsg;
+    }
+}  // namespace
+
+int main(int argc, char* argv[])
+{
+    rclcpp::init(argc, argv);
+
+    rclcpp::spin(std::make_shared<AntennaNode>());
+
+    rclcpp::shutdown();
+    return 0;
+}
+
+AntennaNode::AntennaNode():
+    rclcpp::Node("antenna"),
+    _driver(PUBLISHER_PERIOD_MS)
+{
+    _pubAntennaStatus = this->create_publisher<rover_msgs::msg::AntennaStatus>(TOPIC_ANTENNA_STATUS, QOS_DEFAULT);
+
+    if (loadUserInfo())
+    {
+        sCommandResult loginResult = _driver.setUserInfo(_username, _password);
+        RCLCPP_INFO(this->get_logger(), loginResult.error.c_str());
```

Comment:
Correct syntaxt would be: 
```suggestion
        RCLCPP_INFO(this->get_logger(), "%s", loginResult.error.c_str());
```

Apply throughout